parsec/core/fs/exceptions.py

```diff
@@ -10,16 +10,19 @@
         +-- FSRemoteOperationError
 
 """
+from __future__ import annotations
 
 import os
 import errno
 import io
 
-from parsec.core.types import EntryID, ChunkID, AnyPath
+from parsec.core.types import EntryID, ChunkID
 from parsec.core.fs.utils import ntstatus
 
-from typing import Optional, Union
+from typing import Optional, Union, TYPE_CHECKING
 
+if TYPE_CHECKING:
+    from parsec.core.fs.path import AnyPath
```

Comment:
why ? (I guess it's recursive import related, but would be good to add a short `# Avoid recursive import` comment given `if TYPE_CHECKING` is quite exotic in the codebase)

Reply:
Yes recursive imports. `FsPath` raises an `FsOperationError` that takes an `Union[FsPath, str]` as argument in its `__init__` method.

Reply:
I'll add a comment.

Reply:
65c8edbf94a88609907f7dbec4c55e7dd9c31ca3